ledger-core/network/servicenetwork/watermill.go

```diff
@@ -99,11 +99,3 @@ func (n *ServiceNetwork) processIncoming(ctx context.Context, args []byte) ([]by
 
 	return ack, nil
 }
-
-func getIncomingTopic(msg *message.Message) string {
```

Comment:
only one subscriber topic for watermill left?

Reply:
yep. Second was created, when we had ContractRequester